tests/avocado/test_nec-usb-xhci.py

```diff
@@ -23,6 +23,10 @@ def setUp(self):
 
         # Fedora Cloud Image has default username 'fedora'
         self.vm.add_image(self.image.path, 'fedora', 'fedora', snapshot=False)
+
+        # Fedora Cloud Image has no password for user fedora. Calling
+        # cloudinit() to set the password as defined in add_image()
+        self.vm.cloudinit()
```

Comment:
Again, an obvious improvement.  Still, the challenge is to remove as much boilerplate code from tests as possible (assuming we take as the default to prepare the image with cloudinit).

Reply:
That will be addresses when I change the `cloudinit()` to be a parameter of `add_image()`. Less boiler plate will be here with that change.